src/pudl/metadata/resources/eia861.py

```diff
@@ -224,16 +228,16 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_demand_side_management_sales": {
-        "description": (
-            """Electricity sales for resale and to ultimate customer.
-
-The raw EIA861 demand-side management (DSM) table (split into three normalized tables in
+        "description": {
+            "additional_summary_text": "electricity sales related to demand-side management (DSM).",
```

Comment:
Will users expect searching for the old "resale" "ultimate customer" vocab to match this table? if so we should find a way to put it back  in